components/org.wso2.carbon.identity.application.authenticator.biometric.servlet/src/main/java/org/wso2/carbon/identity/application/authenticator/biometric/servlet/model/WaitStatus.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+
+package org.wso2.carbon.identity.application.authenticator.biometric.servlet.model;
+
+/**
+ * This class represents the status response for status and challenge returned from the android device.
```

Comment:
`..from the android device..`

Let's refactor `android` as `mobile`

Reply:
done

Reply:
still there in the PR

Reply:
Check the latest commit